script.js

```diff
@@ -2,3 +2,83 @@ var main = function (input) {
   var myOutputValue = 'hello world';
   return myOutputValue;
 };
+// set global var//
+var win = 0;
+var lose = 0;
+var draw = 0;
+var troll = 0;
+
+// hack function//
+var Hack = function (input) {
```

Comment:
Why do you have 2 functions called `Hack`? What do these do? I see you're trying to build in cheat codes into your game which is cool, but while the cheat codes can be mysterious to a user, they should NOT be mysterious in the code logic. Please name your function more clearly, e.g. `activateReverseMode` instead of `Hack` which is very confusing.